common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -252,6 +253,36 @@ def test_can_show_answer(self):
         self.assertTrue(self.problem_page.is_focus_on_problem_meta())
 
     @attr(shard=7)
+    def test_save_reaction(self):
+        """
+        Scenario: Verify that the save button performs as expected with problem types
+
+        Given that I am on a problem page
+        And I can see a CAPA problem with the Save button present
+        When I select and answer and click the "Save" button
+        Then I should see the Save notification
+        And the Save button should not be disabled
+        And if I change the answer selected
+        Then the "Save" button should be enabled
+        And the Save notification should be removed
+        """
+        self.problem_page.wait_for(
+            lambda: self.problem_page.problem_name == self.problem_name,
+            "Make sure the correct problem is on the page"
+        )
+        self.problem_page.wait_for_page()
+        self.answer_problem(correctness='correct')
+        self.assertTrue(self.problem_page.is_save_button_enabled())
+        self.problem_page.click_save()
+        self.problem_page.wait_for_save_notification()
+        self.assertTrue(self.problem_page.is_save_button_enabled())
+        self.answer_problem(correctness='incorrect')
+        self.assertTrue(self.problem_page.is_save_button_enabled())
```

Comment:
I will take another look at this, it is still useful before and after saving to ensure the button is disabled during saving and then reenabled.